.github/workflows/clang_tidy_diff_review.yml

```diff
@@ -0,0 +1,37 @@
+name: ClangTidyDiffReview
+
+# You can be more specific, but it currently only works on pull requests
+on: [pull_request]
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: true
+
+    # Optionally generate compile_commands.json
+
+    - name: Checkout submodules
+      run: git submodule update --init --recursive
+
+    - uses: ZedThree/clang-tidy-review@v0.14.0
```

Comment:
I suspect the [split workflow](https://github.com/marketplace/actions/clang-tidy-review#usage-in-fork-environments-split-workflow) should be implemented to handle PRs from forked repos.

Reply:
I have applied the changes fort split_workflow.